CFC_DataCollector/recommender/recommendation_pipeline.py

```diff
@@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
         return []
 
     def get_selected_user_utility_model(self, user_id):
-        return []
+        return UserUtilityModel.find_from_db(user_id)
 
     def recommend_trips(self, trip_id, utility_model):
         return []
```

Comment:
Ok, thanks. Multiplying probabilities doesn't quite make sense with our
model... Can you send me vijs? Thanks. As far as production, early next
week (Monday?) should be fine, especially if other students are available
to help.

On Wednesday, April 29, 2015, shankari <notifications@github.com> wrote:

> In CFC_DataCollector/recommender/recommendation_pipeline.py
> <https://github.com/e-mission/e-mission-server/pull/111#discussion_r29350629>
> :
>
> > @@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
> >          return []
> >
> >      def get_selected_user_utility_model(self, user_id):
> > -        return []
> > +        return UserUtilityModel.find_from_db(user_id)
> >
> >      def recommend_trips(self, trip_id, utility_model):
> >          return []
>
> Let me send you Vij's code and we can see how easy it is to use.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/111/files#r29350629>.
>